diff -ru stock/dm.xmlsec.binding-1.3.2/setup.py mod/dm.xmlsec.binding-1.3.2/setup.py
--- stock/dm.xmlsec.binding-1.3.2/setup.py  2015-01-10 03:18:26.000000000 -0500
+++ mod/dm.xmlsec.binding-1.3.2/setup.py    2015-07-10 15:43:41.000000000 -0400
@@ -95,19 +95,22 @@
 if libxml2_libs[:2] not in ["-l", "-L"]:
     sys.exit("Error : cannot get LibXML2 linker flags; do you have the `libxml2` development package installed?")
 
+XMLSEC_CONFIG=environ.get('XMLSEC_CONFIG', '')
 crypto_engine = environ.get("XMLSEC_CRYPTO_ENGINE")
 if crypto_engine is None:
-  crypto_engine = commands.getoutput("xmlsec1-config --crypto")
+  crypto_engine = commands.getoutput("xmlsec1-config %s --crypto" % XMLSEC_CONFIG)
   if not crypto_engine:
     sys.exit("Error: cannot get XMLSec1 crypto engine")
 else:
   assert crypto_engine in ("openssl", "gnutls", "nss")
-crypto_engine = " --crypto=" + crypto_engine
-xmlsec1_cflags = commands.getoutput("xmlsec1-config --cflags" + crypto_engine)
+XMLSEC_CONFIG += " --crypto=" + crypto_engine
+xmlsec1_cflags = commands.getoutput("xmlsec1-config %s --cflags" % XMLSEC_CONFIG)
+print XMLSEC_CONFIG
+print environ
 if xmlsec1_cflags[:2] not in ["-I", "-D"]:
     sys.exit("Error: cannot get XMLSec1 pre-processor and compiler flags; do you have the `libxmlsec1` development package installed?")
 
-xmlsec1_libs = commands.getoutput("xmlsec1-config --libs" + crypto_engine)
+xmlsec1_libs = commands.getoutput("xmlsec1-config %s --libs" % XMLSEC_CONFIG)
 if xmlsec1_libs[:2] not in ["-l", "-L"]:
     sys.exit("Error : cannot get XMLSec1 linker flags; do you have the `libxmlsec1` development package installed?")
 
